hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -44,6 +44,7 @@ def step(self, verbose: bool) -> Tuple[Optional[OTBNInsn], List[str]]:
             return (None, [])
 
         was_stalled = self.state.stalled
+        old_pc = self.state.pc
```

Comment:
So that's less of the "old" PC, but more the "current" PC? Maybe rename it to make it slightly clearer?

Reply:
Hmm. But it **is** the "old" PC by the time the variable is read :-) I'll switch to "pc_before", which should hopefully be clearer.

Reply:
Yeah, but when looking at the function, I was thinking about the time step (cycle) when the message is printed, not so much about the variable order. So "old" is "previous cycle" in my mind. But no strong feelings, whatever you're comfortable with.